ConfigurationSystem/Client/CSAPI.py

```diff
@@ -411,12 +411,12 @@ def addShifter( self, shifters = None ):
     :return: S_OK/S_ERROR
     """
 
-    def getOpsSection( ):
+    def getOpsSection():
```

Comment:
Actually, I disagree... In this case, the exception is perfectly suitable. If you look where the try/except clause is, it makes perfect sense to use that.